TSC_Minutes/20231109_TSC_MOM.md

```diff
@@ -0,0 +1,48 @@
+# SEAPATH -  TSC Meeting
+
+* Date: 12th October 2023
+* Attendees: 
+    - Eloi Bail (Savoir-faire Linux)
+    - Alban Denat (Schneider)
+    - Mathieu Dupré (Savoir-faire Linux)
+    - Anton Krupskii (Welotec)
+    - Paul Le Guen de Kerneizon (Savoir-faire Linux)
+    - David McDonald (GE)
+    - Laurent Morin (Schneider)
+    - Erwann Roussy (Savoir-faire Linux)
+    - Jan Hille (Welotec)
+    - Sander Jansen (Alliander)
+    - ben van 't ende (Alliander)
+
+# Agenda
+
+- Presentation of LFEnergy Compas
+- Discussion on cybersecurity
+- Agenda for next TSC
+
+## Discussion
+
+- Presentation of LFEnergy Compas
+- IEC weeks:
+    - 150 attendees
+    - utilities
+    - TSO / DSO
+    - 2 topics:
+        - worried with vendor-lockin (ex: Vmware)
+        - VPAC
+            - Interrested to implement virtualized protection
+    - Could be interresting to showcase SEAPATH next IEC weeks
+
+- Presentation of IEC62443 
+- Presentation available at:
+    - https://github.com/com-pas/contributing/blob/main/presentations/231109%20CoMPAS%20introduction%20for%20Seapath.pptx
+    - https://github.com/seapath/TSC/blob/master/TSC_Minutes/20231012_WelotecIEC62443.pdf
+
```

Comment:
The presentation of the IEC 62443-4-2 is missing.

Reply:
good catch